pkg/rancher-desktop/components/WSLProxy.vue

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+import { Checkbox } from '@rancher/components';
+import _ from 'lodash';
+import Vue from 'vue';
+
+import RdFieldset from '@pkg/components/form/RdFieldset.vue';
+import { Settings } from '@pkg/config/settings';
+import { RecursiveTypes } from '@pkg/utils/typeUtils';
+
+import type { PropType } from 'vue';
+
+export default Vue.extend({
+  name: 'wsl-proxy',
+
+  components: { Checkbox, RdFieldset },
+
+  props: {
+    description: {
+      type:    String,
+      default: '',
+    },
+    emitChanges: {
+      type:    Boolean,
+      default: false,
+    },
```

Comment:
I'm not seeing `emitChanges` used anywhere. I think this existed solely for consumption by the first run dialog.

If this is the case, I think we should remove this prop because it will simplify the `onChange` method.